projects/core/components/root/root.component.ts

```diff
@@ -63,6 +63,6 @@ export class TuiRootComponent {
         @Inject(DOCUMENT) {body}: Document,
         @Inject(TUI_THEME) theme: string,
     ) {
-        body.setAttribute('data-tui-theme', theme.toLowerCase());
+        body.parentElement?.setAttribute('data-tui-theme', theme.toLowerCase());
```

Comment:
Instead of `body` use `documentElement`.

Reply:
fixed